fa_crosshair/init.sqf

```diff
@@ -67,32 +76,32 @@ dc_fac_fnc_crosshairInit = {
 				"AbdvPercMstpSnonWrfl",
 				"AsdvPercMstpSnonWrfl"
 			];
-			
+
 			// Only show crosshair if player alive, player not aiming down sights, animation correct and vanilla crosshair off
 			if (alive player && cameraView in ["INTERNAL","EXTERNAL"] && _showCrosshair && difficultyOption "weaponCrosshair" == 0) then {

```

Comment:
crosshairEnable should be here. Though there's an argument for moving everything into the _showCrosshair bool instead of whatever I've done.